### PR TITLE
Feat: enhance velq ql and support cue file

### PR DESCRIPTION
**Pull request by wonderflow**
Signed-off-by: Jianbo Sun <jianbo.sjb@alibaba-inc.com>

### Description of your changes

This PR changes the experience of vela ql, supports:

```
vela ql --query "<query>"
vela ql --file my-view.cue
```

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by barnettZQG** [comment]
@wonderflow `--file` provide a view definition? If yes, how to provide the parameters?
Or:
```
vela ql --query xxx --view-file=./view.cue
```

**Comment by wonderflow** [comment]
> @wonderflow `--file` provide a view definition? If yes, how to provide the parameters? Or:
> 
> ```
> vela ql --query xxx --view-file=./view.cue
> ```

user can write the parameter in the cue file, actually we don't need any parameter as the CUE file is concrete. Use case:

```
vela ql --view-file=./view.cue
```

The view file should be like:

```
import (
	"vela/ql"
)
configmap: ql.#Read & {
   value: {
      kind: "ConfigMap"
      apiVersion: "v1"
      metadata: {
        name: "mycm"
      }
   }
}
status: configmap.value.data.key

export: "status"
```

File: references/cli/velaql.go [hunk 1]
```diff
@@ -42,39 +42,87 @@ type Filter struct {
...
+				return queryFromStatement(ctx, newClient, c, querySts, cmd)
+			}
+			return queryFromView(ctx, newClient, c, cueFile, cmd)
 		},
 		Annotations: map[string]string{
 			types.TagCommandOrder: order,
 			types.TagCommandType:  types.TypeApp,
 		},
 	}
+	cmd.Flags().StringVarP(&cueFile, "file", "f", "", "The CUE file path for VelaQL.")
```
Comment (barnettZQG) [inline references/cli/velaql.go:93]:
```suggestion
	cmd.Flags().StringVarP(&cueFile, "view-file", "f", "", "The CUE file path for VelaQL.")
```

Comment (wonderflow) [reply references/cli/velaql.go:93]: no, this is too long for an args

File: references/cli/velaql.go [hunk 2]
```diff
@@ -42,39 +42,92 @@ type Filter struct {
...
-		Short:   "Show result of executing velaQL.",
-		Long:    "Show result of executing velaQL.",
-		Example: `vela ql "view{parameter=value1,parameter=value2}"`,
+		Use:   "ql",
+		Short: "Show result of executing velaQL.",
+		Long:  "Show result of executing velaQL.",
+		Example: `Users can query with a query statement:
+		vela ql --query "<inner-view-name>{<param1>=<value1>,<param2>=<value2>}"
+They can also query by a ql file:
+		vela ql --view-file=./ql.cue
```
Comment (barnettZQG) [inline references/cli/velaql.go:54]:
```suggestion
		vela ql --file=./ql.cue
```

Comment (wonderflow) [reply references/cli/velaql.go:54]: Good catch

**Review by Somefive — APPROVED** [review]

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**